ge08_team01/views/motorcycle_registry_portal_templates.xml

```diff
@@ -0,0 +1,204 @@
+                </t>
+            </xpath>
+        </template>
+
+        <template id="portal_my_motorcycle_registries_list" name="My Motorcycle Registries">
+            <t t-call="portal.portal_layout">
+                <t t-set="breadcrumbs_searchbar" t-value="True" />
+
+                <t t-call="portal.portal_searchbar">
+                    <t t-set="title">Motorcycle Registries</t>
+                </t>
+                <t t-call="portal.portal_table">
+                    <thead>
+                        <tr class="active">
+                            <th>Registry #</th>
+                            <th>VIN</th>
+                            <th>Make</th>
+                            <th>Model</th>
+                            <th>License Plate Number</th>
+                            <th>Owner</th>
+                            <th>Lot ID</th>
+                        </tr>
+                    </thead>
+                    <t t-foreach="registries" t-as="registry">
+                        <tr>
+                            <td>
+                                <a t-att-href="registry.get_portal_url()">
+                                    <t t-out="registry.registry_number" />
+                                </a>
+                            </td>
+                            <td>
+                                <span t-field="registry.vin" />
+                            </td>
+                            <td>
+                                <span t-field="registry.make" />
+                            </td>
+                            <td>
+                                <span t-field="registry.model" />
+                            </td>
+                            <td>
+                                <span t-field="registry.license_plate" />
+                            </td>
+                            <td>
+                                <span t-field="registry.owner_id" />
+                            </td>
+                            <td>
+                                <span t-field="registry.lot_id" />
+                            </td>
+                        </tr>
+                    </t>
+                </t>
+            </t>
+        </template>
+
+        <template id="motorcycle_registry_portal_template"
+            name="Motorcycle Registry Portal Template" inherit_id="portal.portal_sidebar"
+            primary="True">
+            <xpath expr="//div[hasclass('o_portal_sidebar')]" position="inside">
+                <t t-set="custom_html">This is the portal view of your Motorcycle Registry.</t>
+                <t t-if="registry.owner_id == env.user.partner_id"
+                    t-set="o_portal_fullwidth_alert"
+                    groups="motorcycle_registry.group_registry_user">
+                    <t t-call="portal.portal_back_in_edit_mode" />
+                </t>
+
+                <div class="row mt16 o_portal_sidebar">
+                    <!-- Sidebar -->
```

Comment:
You do not really need the comment, the class makes it pretty clear

Reply:
Same x)